### ISSUE TITLE
auto-link posts that are duplicates of this one

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#1348]
meta:290798

When a top-level post is closed as a duplicate, that post shows a link to the duplicate target.  This request is for the reverse: if a post is the target of one or more duplicates, please show those links on the post page somehow.  I wouldn't use a notice in the body of the post (that would look too much like the close notice), but maybe we could add "duplicated by: (links)" below the post and above the comments, or as a special comment thread, or something else TBD.

**Comment by cellio** [comment]
I was thinking about this again today, and I think this would help with connecting related posts to each other -- sometimes there's some additional info in one of the duplicates that doesn't get edited into the target at the time, and seeing the list could help us pull information together.  A "Duplicated by: (links)" block under the post and above the comment threads would suffice.

**Comment by ArtOfCode-** [comment]
What if we added this to the sidebar?

<img width="1138" height="765" alt="Image" src="https://github.com/user-attachments/assets/595d9a63-7e16-4626-8248-e59860939f0b" />

**Comment by Oaphi** [comment]
Agreed, but I'd put them under the featured / hot section - makes more sense to me to give priority to featured & hot posts

**Comment by ArtOfCode-** [comment]
I debated that, not sure which I prefer

**Comment by cellio** [comment]
And on the other hand, it's also reasonable to say that stuff about "this post" is higher priority than other stuff about the community, so it should go first as proposed in the screenshot.  Especially if we're going to use color to distinguish it (our "notice" formatting), that might make more sense.

**Comment by Oaphi** [comment]
>  it's also reasonable to say that stuff about "this post" is higher priority than other stuff about the community

Local is not always of higher priority than global 😅 (even though it often is). Compromising on splitting the widgets and keeping "features" at the top, then "related" (or "linked", etc), then "hot" works for me, though - @ArtOfCode- ?

**Comment by ArtOfCode-** [comment]
Works for me
